samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/Configure.cshtml

```diff
@@ -22,45 +23,47 @@
 </p>
 <script>
 
-    microsoftTeams.initialize();
-    var conextchatId = "";
-    // Check the initial theme user chose and respect it
-    microsoftTeams.getContext(function (context) {
-        conextchatId = context.chatId;
-        console.log(context);
-        if (conextchatId != "") {
-            document.getElementById("textConfiguration").innerHTML = "Configure your Group Chat Tab to Send Activity Feed Notification: ";
-             $("#teamsTab").hide();
-             $("#groupChatTab").show();
-        }
-        else {
-            document.getElementById("textConfiguration").innerHTML = "Configure your Teams Tab to Send Activity Feed Notification: ";
-             $("#groupChatTab").hide();
-             $("#teamsTab").show();
-        }
-        if (context && context.theme) {
-            setTheme(context.theme);
-        }
-    });
+    microsoftTeams.app.initialize().then(() => {
+        var conextchatId = "";
+        // Check the initial theme user chose and respect it
+        microsoftTeams.getContext(function (context) {
+            conextchatId = context.chatId;
 
-    // Handle theme changes
-    microsoftTeams.registerOnThemeChangeHandler(function (theme) {
-        setTheme(theme);
-    });
+            if (conextchatId != "") {
+                document.getElementById("textConfiguration").innerHTML = "Configure your Group Chat Tab to Send Activity Feed Notification: ";
+                $("#teamsTab").hide();
+                $("#groupChatTab").show();
+            }
+            else {
+                document.getElementById("textConfiguration").innerHTML = "Configure your Teams Tab to Send Activity Feed Notification: ";
+                $("#groupChatTab").hide();
+                $("#teamsTab").show();
+            }
 
-    // Save configuration changes
-    microsoftTeams.settings.registerOnSaveHandler(function (saveEvent) {
-        // Let the Microsoft Teams platform know what you want to load based on
-        // what the user configured on this page
-        microsoftTeams.settings.setSettings({
-            contentUrl: createTabUrl(), // Mandatory parameter
-            entityId: createTabUrl() // Mandatory parameter
+            if (context && context.theme) {
+                setTheme(context.theme);
+            }
         });
 
-        // Tells Microsoft Teams platform that we are done saving our settings. Microsoft Teams waits
-        // for the app to call this API before it dismisses the dialog. If the wait times out, you will
-        // see an error indicating that the configuration settings could not be saved.
-        saveEvent.notifySuccess();
+        // Handle theme changes
+        microsoftTeams.registerOnThemeChangeHandler(function (theme) {
```

Comment:
shouldn't this be `microsoftTeams.app.registerOnThemeChangeHandler` in v2? https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=Json-v2%2Cteamsjs-v2#handle-theme-change

Reply:
changes this to v2 implementation